setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal = 1
```

Comment:
Since all our code is python2/3 compatible we can actually make a single universal wheel. I have tested this in rAV and it seems to work fine